docs/UserGuide.md

```diff
@@ -174,7 +174,11 @@ Format: `delete INDEX`
 Example: `delete 3` deletes the 3rd task in Harmonia.
 
 <div markdown="span" class="alert alert-primary">:bulb: **Note:**
-You can key `list` to see the sequence of tasks in the list to check the index of the task you wish to delete.
+You can key `list` to check the index of the task you wish to delete.
+</div>
+
+<div markdown="span" class="alert alert-primary">:warning: **Note:**
+This action is irreversible and all deleted tasks cannot be retrieved.
 </div>
```

Comment:
Should a warning note follow this formatting: `<div markdown="span" class="alert alert-warning">:exclamation: **Warning:** This is an example warning</div>`?